### PR TITLE
Stories/dt/template #3074

### PR DESCRIPTION
**Pull request by terw-dan**
Fixes https://github.com/Yoast/wordpress-seo/issues/3074
Adds checks in the app to use templates on strings for the snippet.

### CODE REVIEW

**Comment by atimmer** [comment]
Please re-issue this PR on master

File: dist/yoast-seo.js [hunk 1]
```diff
@@ -1113,10 +1114,21 @@ YoastSEO.App.prototype.constructI18n = function( translations ) {
 };
 
 /**
+ * Retrieves data from the callbacks.getData and applies modification to store these in this.rawData.
+ */
+YoastSEO.App.prototype.getData = function() {
+	this.rawData = this.callbacks.getData();
+	if ( this.pluggable.loaded ) {
+		this.rawData.pageTitle = this.pluggable._applyModifications( "data_pageTitle", this.rawData.pageTitle );
```
Comment (atimmer) [inline dist/yoast-seo.js]: We shouldn't be combining 🐍-case and 🐫-case.

File: js/snippetPreview.js [hunk 2]
```diff
@@ -335,30 +329,6 @@ YoastSEO.SnippetPreview.prototype.setUnformattedText = function( ev ) {
 };
 
 /**
- * Adds the siteName to the snippetTitle when editting starts, this adds the sitename to
- * the snippet_sitename that cannot be edited.
- * Sets the display property to inline-block of the snippet_title so we can set a width.
- * @param ev
- */
-YoastSEO.SnippetPreview.prototype.setSiteName = function( ev ) {
```
Comment (atimmer) [inline js/snippetPreview.js]: Can you put in the CHANGELING.md that we deleted these since RC-1.

**Comment by terw-dan** [comment]
processed codereview

**Comment by omarreiss** [comment]
Merged in 9658393a

**Pull request closed without merging**